site/en/blog/permissions-chip/index.md

```diff
@@ -68,16 +65,17 @@ how Windows users reacted to the geolocation and the notification prompts on sit
 Given an ignore/dismiss rate of approximately 85%, and especially given how much the prompt stands out and
 insists on users making a decision immediately, there is a conflict between the level of urgency
 assumed by the browser versus the user's preference for waiting to make a decision. This creates the
-perception that it's "annoying" for a site to ask for a permission.
+perception that it's "annoying" for a site to ask for a permission, apart from potential extra
+things users need to react to like cookie consent banners, newsletter signups, etc.
```

Comment:
I don't understand what the added text is trying to tell me. Please explain in more words so I can make a suggestion.

Reply:
When you land on a page you have to deal with the notifications prompt, then with the cookie banner, then the newsletter signup,… It's many things for just reading an article.

Reply:
I think I understand. Permissions get lost because of the general site noise.

Reply:
Lost as in clicked away without even being read and thus contributing to the general perception of the web being annoying.

Reply:
I just made a suggestion.